examples/spark/spark-driver.json

```diff
@@ -11,7 +11,7 @@
     "containers": [
       {
         "name": "spark-driver",
-        "image": "gurvin/spark-driver",
+        "image": "gcr.io/google_containers/spark-driver",
```

Comment:
Versioning?  We'll need to keep cadence with upstream and calling out versions is important here.